front/app/containers/Admin/tools/PublicAPI/PublicAPITokens/CreateTokenModal/index.tsx

```diff
@@ -18,6 +18,7 @@ import {
 import useAddApiClient from 'api/api_clients/useAddApiClient';
 import { handleHookFormSubmissionError } from 'utils/errorUtils';
 import styled from 'styled-components';
+import Warning from "../../../../../../components/UI/Warning";
```

Comment:
Minor: I think we can use just this instead:
`import Warning from "components/UI/Warning";`